site/content/blog/2022-08-25-zap-on-raspberry-pi/index.md

```diff
@@ -0,0 +1,179 @@
+---
+title: "Running ZAP on a raspberry pi"
+summary: >
+  Setting up ZAP on the raspberry pi.
+images:
+- https://www.zaproxy.org/blog/2022-08-25-zap-on-raspberry-pi/images/Screenshot_2022-08-20-18-01-55-325_com.realvnc.viewer.android.jpg
+type: post
+tags:
+- blog
+- gsoc
+date: "2022-08-25"
+authors: 
+    - eingengraou
+---
+
+# Running ZAP on on a raspberry pi
+
+
+## The whole idea !
+I know maybe a lot of people that are going to read this are probably going to ask, ***why go through this stress? , why not just use your computer?***
+
+
+The single reason was that i had issues with steady power supply and needed something to hack with all the time even if i'm away from home so it would be possible to spend more time on targets.
+
+
+Now that you know the reason behind the whole idea , we can dive right in.
+
+
+
+
+## Getting started 
+ Disclaimer: There's currently no method to run [zap](https://github.com/zaproxy/zaproxy) on an android device alone.
```

Comment:
"ZAP" should always be capitalized, unless it is part of a filename such as `zap.sh`